test/cpp.cc

```diff
@@ -6,9 +6,15 @@
  * Authors: Keith Busch <keith.busch@wdc.com>
  */
 
+#include <algorithm>
 #include <iostream>
 #include <libnvme.h>
 
+static int min_compile_test()
+{
+	return std::min(1, 2);
+}
```

Comment:
My point is that the whole change to `test/cpp.c` should be in a different patch. I don't mind if you add the `min` and `max` to catch build errors, just in a separate patch.

Reply:
got it.